src/index.ts

```diff
@@ -243,11 +245,15 @@ export {
   getAltProfileUrlAllFrom,
   getWebIdDataset,
 } from "./profile/webid";
-export {
-  getJsonLdParser,
-  getTurtleParser,
-  solidDatasetAsTurtle,
-} from "./formats/index";
+
+// Export the different formats methods from @inrupt/solid-client import.
+//
+// NOTE: We have to export like this, otherwise rollup does some sort of
+// tree-shaking and breaks the build where `dist/formats/index.mjs` is no longer
+// built, and this causes the `@inrupt/solid-client/formats` subpath export to
+// no longer work.
+export const { getJsonLdParser, getTurtleParser, solidDatasetAsTurtle } =
```

Comment:
These were left out of the public API on purpose, as it is all still quite experimental/not something we want to commit to. Is it possible to add a note to this intent here, in addition to this not being documented?

Reply:
The parsers are still not documented (only solidDatasetAsTurtle is currently), so there's no change in public API, but we need to export them such that our other packages can use them

Reply:
Them being exported, making it extra explicit that they aren't public API wouldn't hurt I think.

Reply:
So like, just add a comment above saying `// Non-public APIs needed by the other SDKs, these are not part of our public supported API` ?

Reply:
Have improved the comment.